pxe/common_files/disk_imaging.sh

```
@@ -818,7 +818,7 @@ function main() {
 		if [ ! -z "$line" ]; then
 			distro_release=$(echo $line|cut -d'=' -f2)
 		fi
-		line=$(echo $i|grep 'ds=nocloud-net;s')
+		line=$(echo $i|grep 'ds=nocloud')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)